tests/testthat/test-mail.R

```diff
@@ -0,0 +1,13 @@
+tools_path <- test_path("testdata", "tools.yml")
+conf_path <- test_path("testdata", "thrift.yml")
+
+test_that("Calling parse_mbox with correct perceval and mbox path returns a data table with correct raw data", {
+  tools_path <- file.path(tools_path)
+  tool <- yaml::read_yaml(tools_path)
+  perceval_path <- tool[["perceval"]]
+  mbox_path <- example_mbox_normal <- function(folder_path = "/tmp", folder_name="sample_folder")  # Replace with the actual path to your mbox file
+  result <- parse_mbox(perceval_path, mbox_path)
+  expect_is(result, "data.table")
```

Comment:
You should use expect_equal several times, and check the value you put on the raw data match exactly here. expect_is is too vague. You may get an empty table and it will pass!